users/src/main/java/io/crate/operation/auth/AuthenticationService.java

```diff
@@ -38,9 +43,26 @@
     private static final String KEY_METHOD = "method";
 
     private final Map<String, AuthenticationMethod> authMethodRegistry = new HashMap<>();
+
+    public Map<String, Map<String, String>> getHbaConf() {
```

Comment:
this method is only used for testing, right? please make it package private and annotate with `@VisibleForTesting`